host/windows/runcli.bat

```diff
@@ -1,17 +1,19 @@
 @ECHO OFF
 
 :: Runs JFrog CLI with the given arguments. The JFrog CLI's home directory is the extension's home directory.
-
-set HOME_DIR=%USERPROFILE%\.jfrog-docker-desktop-extension
-set LOGS_DIR=%HOME_DIR%\logs
-
-set JFROG_CLI_HOME_DIR=%HOME_DIR%
+set "HOME_DIR=%USERPROFILE%\.jfrog-docker-desktop-extension"
```

Comment:
I think thq quotes should be on the variable itself - not the entire SET:
```suggestion
set HOME_DIR="%USERPROFILE%\.jfrog-docker-desktop-extension"
```

Reply:
okay. will check when I have the chance if this change works

Reply:
also no sadly 🥲